api/cpp/esp-idf/slint/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+# Copyright © SixtyFPS GmbH <info@slint.dev>
+# SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
+
+idf_component_register(
+  SRCS "src/esp_slint.cpp"
+  INCLUDE_DIRS "include"
+  REQUIRES "esp_lcd" "esp_lcd_touch"
+  PRIV_REQUIRES "esp_timer")
+
+set(Rust_CARGO_TARGET "xtensa-${IDF_TARGET}-none-elf")
+set(CORROSION_NO_WARN_PARSE_TARGET_TRIPLE_FAILED ON)
```

Comment:
Is this still needed with the latest release?

Reply:
Not until corrosion 0.4.3 that will ship with https://github.com/corrosion-rs/corrosion/pull/425